src/vshaxe/configuration/HaxeInstallation.hx

```diff
@@ -96,7 +96,7 @@ class HaxeInstallation {
 		if (libraryBasePath == null || !FileSystem.exists(libraryBasePath) || !FileSystem.isDirectory(libraryBasePath)) {
 			return [];
 		}
-		return FileSystem.readDirectory(libraryBasePath).filter(file -> !file.startsWith(".")).map(file -> {name: file});
+		return FileSystem.readDirectory(libraryBasePath).filter(file -> !file.startsWith(".")).map(file -> {name: StringTools.replace(file, ',', '.')});
```

Comment:
StringTools should be in import.hx, so you can use it as a static extension.

Reply:
:P I probably have spotted StringTools.startWith is used on the same line

Changed